components/ui-api-layer/docs/resource-quotas-status.md

```diff
@@ -0,0 +1,116 @@
+# ResourceQuotasStatus 
+
+[ResourceQuota](https://kubernetes.io/docs/concepts/policy/resource-quotas/) provides constraints that limit aggregate resource consumption per Namespace.
+
+The ResourceQuota values can be as follows:
+```bash
+Name:            kyma-default
+Resource         Used       Hard
+--------         ---        ---
+limits.memory    7348440Ki  10Gi
+requests.memory  3727576Ki  7Gi
+```
+The `used` values of the ResourceQuota can exceed the `hard` values and block any resource creation in the Environment.
+
+The ResourceQuotasStatus contains the flag and list of exceeded ResourceQuotas limits, together with a set of resources which exceed that limits. The value of the flag informs you if any ResourceQuota is exceeded in any possible way.
+
+ResourceQuotasStatus should detect whenever any ReplicaSet or StatefulSet in your Environment cannot progress because it's blocked by the ResourceQuota.
+To check that you must calculate the required amount of resources to create another replica. If there is not enough resources, you should return the exceeded status.
```

Comment:
``To check if any ReplicaSet or StatefulSet is blocked, calculate the required number of resources to create another replica. If there are not enough resources to create another replica, return the ResourceQuotaStatus with the flag set to `true`.``